practical08/src/store/actions.js

```diff
@@ -0,0 +1,117 @@
+import axios from "axios";
+import routes from "../router/index";
+import cookies from "vue-cookies";
+import jwt_decode from "jwt-decode";
+
+export default {
+  async fetchCars({ commit }) {
+    commit("CHANGE_LOADER", true);
+    await axios
+      .get(`https://testapi.io/api/dartya/resource/cardata`)
+      .then((response) => {
+        commit("SET_CARS", response.data.data);
+        commit("CHANGE_LOADER", false);
+      })
+      .catch((error) => {
+        console.log(error);
```

Comment:
Add toast message here when any error occurs. Do this for all api call avoid alert for this